### PR TITLE
feat: build homepage

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview:
Screenshot (111)
Screenshot (112)
Screenshot (113)
Screenshot (114)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        <a href="http://" class="link-no-style nav-link nav-icon-link">
+          <i class="fa-regular fa-user"></i>
+          <span class="small-fontsize">Login</span>
+        </a>
+        <a href="http://" class="link-no-style nav-link nav-icon-link">
+          <i class="fa-regular fa-heart"></i>
+          <span class="small-fontsize">Wishlist</span>
+        </a>
+        <a href="http://" class="link-no-style nav-link nav-icon-link">
+          <img
```
Comment (ckahub) [inline index.html:52]: extra space is wasted here, use pretttier

File: pages/homepage.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  height: 300px;
+  object-fit: cover;
+  margin-top: 1rem;
+}
+
+.image-btn {
+  margin-top: 13rem;
+  margin-left: 8rem;
+  background-color: var(--pink-color);
+  border-radius: 5px;
```
Comment (ckahub) [inline pages/homepage.css:40]: use rem everywhere

**Review by ckahub — CHANGES_REQUESTED** [review]
looks good to me

File: pages/homepage.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  background-image: url("/assets/banner.png");
+  width: 100%;
+  height: 300px;
+  object-fit: cover;
+  margin-top: 1rem;
+}
+
+.image-btn {
+  margin-top: 13rem;
+  margin-left: 8rem;
```
Comment (anupkgurung) [inline pages/homepage.css]: this can be adjusted with shorthand too.

File: pages/homepage.css [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+.active {
+  color: var(--bg-color);
+}
+
+.banner {
+  background-image: url("/assets/banner.png");
+  width: 100%;
+  height: 300px;
+  object-fit: cover;
```
Comment (anupkgurung) [inline pages/homepage.css:33]: why px? and not relative a unit ?

File: pages/homepage.css (hunk 2 above)
Comment (ishitakeshawani) [reply pages/homepage.css:40]: Yes I am using it but Vikram sir told to not to use rem for border and value which is less than 10px.

**Review by anupkgurung — APPROVED** [review]
One suggestion when pushing your code try not to include images except for the screenshot it as there will nothing to review it. You can add them directly to dev or can push it later.

File: index.html (hunk 1 above)
Comment (ishitakeshawani) [reply index.html:52]: I am using it but sometimes it is not working. Do you have any idea of other tools?

**Comment by ishitakeshawani** [comment]
> One suggestion when pushing your code try not to include images except for the screenshot it as there will nothing to review it. You can add them directly to dev or can push it later.

Ok, I will add images later or directly to dev. Thanks.

File: pages/homepage.css (hunk 4 above)
Comment (ishitakeshawani) [reply pages/homepage.css:33]: forgot to use it. Thanks for reminding.

File: pages/homepage.css (hunk 3 above)
Comment (ishitakeshawani) [reply pages/homepage.css]: done

File: pages/homepage.css [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+.category-list-item,
+.top-brands-item {
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 5px;
+  cursor: pointer;
+  width: 15rem;
+  height: 15rem;
+  box-shadow: 0 3px 10px rgb(0 0 0 / 0.2);
```
Comment (Naveenchandar) [inline pages/homepage.css:91]: Please use hex code for consistency.

**Pull request merged**